client/lib/plans/features-list.tsx

```diff
@@ -659,6 +660,26 @@ export const FEATURES_LIST: FeatureList = {
 		},
 	},
 
+	[ FEATURE_CUSTOM_DOMAIN_DEFERRED ]: {
+		getSlug: () => FEATURE_CUSTOM_DOMAIN_DEFERRED,
+		getTitle: () => {
+			return i18n.translate( 'Free domain for one year after first full-price renewal', {
+				context: 'title',
+			} );
+		},
+		getAlternativeTitle: () => i18n.translate( 'Free custom domain after full-price renewal' ),
+		getDescription: () => {
+			return i18n.translate(
+				'Get a custom domain – like {{i}}yourgroovydomain.com{{/i}} – free for the first year after your first full-price renewal.',
```

Comment:
Nit: existing `full price` usages in calypso doesn't have hyphens 
```suggestion
				'Get a custom domain – like {{i}}yourgroovydomain.com{{/i}} – free for the first year after your first full price renewal.',
```